show/main.py

```diff
@@ -1005,12 +1006,18 @@ def platform():
 
 # 'summary' subcommand ("show platform summary")
 @platform.command()
-def summary():
+@click.option('--json', is_flag=True, help="JSON output")
+def summary(json):
     """Show hardware platform information"""
+
     hw_info_dict = get_hw_info_dict()
-    click.echo("Platform: {}".format(hw_info_dict['platform']))
-    click.echo("HwSKU: {}".format(hw_info_dict['hwsku']))
-    click.echo("ASIC: {}".format(hw_info_dict['asic_type']))
+    if json:
+        click.echo(hw_info_dict)
```

Comment:
While this might work in this instance, for the sake of properness, maybe this should read:

```python
click.echo(json.dumps(hw_info_dict))

```